src/PrestaShopBundle/Controller/Admin/Sell/Order/CartController.php

```diff
@@ -438,12 +438,15 @@ public function editProductQuantityAction(Request $request, int $cartId, int $pr
     {
         try {
             $newQty = $request->request->getInt('newQty');
+            $combinationId = $request->request->getInt('attributeId');
```

Comment:
```suggestion
            $attributeId = $request->request->getInt('attributeId');
```

Reply:
No combination matches more with the Combination object and represents more what it actually is, it's a variation of a product which is composed of attributes
Attribute is a characteristic of a product/combination (size, color, ...) In most of the recent command/handlers we used Combination rather than Attribute

If anything it's the parameter which should be renamed rather than the variable here